installer/templates/phx_web/components/core_components.ex

```diff
@@ -366,6 +366,17 @@ defmodule <%= @web_namespace %>.CoreComponents do
     """
   end
 
+  def input(%{type: "hidden"} = assigns) do
+    ~H"""
+    <input
+      type="hidden"
+      name={@name}
+      id={@id}
+      value={Phoenix.HTML.Form.normalize_value(@type, @value)}
+    />
```

Comment:
We probably want a `{@rest}` in here to capture any additional provided attributes.

Reply:
Rest consists of things that should not be in hidden input as well. At least to my understanding. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden

Reply:
Passing something like `form` can still be relevant for hidden inputs, therefore rest should be included in my opinion.

Reply:
Added.